From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ravi Kumar Bokka <brkum@codeaurora.org>
Date: Sun, 29 Oct 2023 12:19:09 +0100
Subject: [PATCH 03/18] sm6225-common: wifi: Remove obsolete McastBcastFilter
 config

Remove obsolete McastBcastFilter ini feature

The McastBcastFilter configuration item is not being used, and support
is being dropped from the driver. Remove the McastBcastFilter setting
feature from the driver.

Ref: https://git.codelinaro.org/clo/la/platform/vendor/qcom-opensource/wlan/qcacld-3.0/-/commit/91a8f3c649af586d356145e57353e257a9d75738

Change-Id: I6fc0e48e31c01ba2df73c51b3fd9ec2a84571541
CRs-Fixed: 2045284
---
 wifi/WCNSS_qcom_cfg.ini | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/wifi/WCNSS_qcom_cfg.ini b/wifi/WCNSS_qcom_cfg.ini
index 7d55372..b3d619d 100644
--- a/wifi/WCNSS_qcom_cfg.ini
+++ b/wifi/WCNSS_qcom_cfg.ini
@@ -32,11 +32,6 @@ InfraUapsdBkSrvIntv=0
 # Flag to allow STA send AddTspec even when ACM is Off
 gAddTSWhenACMIsOff=1
 
-# Flags to filter Mcast and Bcast RX packets.
-# Value 0: No filtering, 1: Filter all Multicast.
-# 2: Filter all Broadcast. 3: Filter all Mcast abd Bcast
-McastBcastFilter=3
-
 #Flag to enable HostARPOffload feature or not
 hostArpOffload=1
 
